resources/js/vat-validation.js

```diff
@@ -118,6 +118,6 @@ function init () {
 }
 
 document.addEventListener('vue:loaded', init)
-if (window.app && window.app.$on) {
+if (window.$on) {
     init()
 }
```

Comment:
This works and is no problem at all, but since this is only for the event listener we can actually do away with the init function, and loaded listener by setting autoremove to false 🚀 

https://github.com/rapidez/core/pull/951/files#diff-4469b4edb873b382f9862dcab91e8b54769fef2e2c3edfb4561fd29c861534cbR14
Example:
```js
import { on } from 'Vendor/rapidez/core/resources/js/polyfills/emit.js'
on('vat-change', async (event) => {
...
}, { autoremove: false })
```